lumicks/pylake/tests/test_kymo.py

```diff
@@ -18,8 +18,9 @@ def test_kymo_properties(h5_file):
                                         [2.084375e+10, 2.187500e+10, 2.284375e+10, 2.387500e+10]], np.int64)
 
         assert repr(kymo) == "Kymo(pixels=5)"
-        assert kymo.has_fluorescence
-        assert not kymo.has_force
+        with pytest.warns(DeprecationWarning):
+            assert kymo.has_fluorescence
+            assert not kymo.has_force
```

Comment:
I would suggest using [`pytest.deprecated_call()`](https://docs.pytest.org/en/stable/warnings.html#ensuring-code-triggers-a-deprecation-warning) for this.

Also, if the intention is to test the deprecation, I believe that each statement requires its own `with` clause since `has_force` could omit the warning if `has_flourescence` already raised one.

Lastly, you could also test if `.json()` also raises the warning.

Reply:
Indeed, each statement needs its own check.

Reply:
👍 I wasn't aware of `pytest.deprecated_call()`, thanks!